package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "devolutions-gateway",
+  "name": "devolutions-jet",
```

Comment:
question: Why is it renamed?

Reply:
My bad. I will revert

Reply:
Wait, I just realized this file should not even exist. It seems it has been commit and merged by mistake :joy:

Reply:
Removed the file